worker/index.js

```diff
@@ -71,8 +71,19 @@ async function work () {
   }
 
   await boss.start()
+
+  const [lastConfirmed] = await models.$queryRaw`SELECT "confirmedIndex" FROM "Invoice" ORDER BY "confirmedIndex" DESC NULLS LAST LIMIT 1`
```

Comment:
This still feels like we're failing to encapsulate this wallet code. Ideally, none of this wallet stuff should be in this file.

If you want I can take a stab at it.

Reply:
Ah, no, I got it. I have an idea.